content/blog/2025-04-07-spring-2025-raspberrypi-updates.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Spring 2025 Raspberry Pi Updates"
+type: blog
+author:
+  name: "Koichiro Iwao"
+  bio: "Engineer at Cybertrust Japan"
+  image: /users/koichiroiwao.jpg
+date: '2025-04-07'
+#images:
+post:
+  title: "Spring 2025 Raspberry Pi Updates"
```

Comment:
Once you add the header image, please also add the below line under the 'title' line here to reference it for the meta data:

`    image: /blog-images/2025/2025.04.08.RaspberryPiUpdates.png`